helm/observability-bundle/templates/apps.yaml

```diff
@@ -19,6 +19,8 @@ spec:
   {{- end }}
   install:
     skipCRDs: {{.skipCRDs | default false }}
+    timeout: 10m
+    upgrade: 10m
```

Comment:
I think you need to set 
```
upgrade:
  timeout: 10m
``` 

at the same level as install like https://docs.giantswarm.io/use-the-api/management-api/crd/apps.application.giantswarm.io/

Reply:
Done